web-client/src/components/CartItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@/components/ui/button';
+import { Card, CardHeader, CardContent, CardFooter } from '@/components/ui/card';
+import { Trash2 } from "lucide-react";
+
+interface CartItemProps {
+    sockId: number;
+    sockVariantId: number;
+    name: string;
+    size: string;
+    price: number;
+    quantity: number;
+    imageUrl: string;
+    onIncrease: () => void;
```

Comment:
Good call on passing/binding the `onIncrease` and other functions to avoid making this component "smart" :)